### PR TITLE
Media sync

### PR DESCRIPTION
**Pull request by Johanndutoit**
Includes a few fixes to Docker scripts and fixes a race condition we found when installing a new router.

Along with that includes the scripts that will download and sync the media library down to the node in a separate process.

### CODE REVIEW

**Comment by bruskiza** [comment]
@Johanndutoit is this ready for review?
@smn ^^^

**Comment by Johanndutoit** [comment]
@bruskiza go for it :)

**Comment by Johanndutoit** [comment]
Interested to hear your thoughts as well

File: scripts/sync.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (bruskiza) [inline scripts/sync.sh:1]: Suggest using a lock file here... to see if it's already running.

Use case:

```
Sync started @ 12am
Now 6am
Next sync starts

Result:
2 sync.sh files running at the same time.
```

Comment (Johanndutoit) [inline scripts/sync.sh:1]: Mmmm well I do kill all instances before the sync.sh is run again on cron. Think a lock file is still needed ?

Comment (Johanndutoit) [inline scripts/sync.sh:1]: See - https://github.com/praekelt/goddard-node-agent/pull/18/files#diff-cdf420c4461eda6072b5c0dbeb193e5eR98

Comment (bruskiza) [inline scripts/sync.sh:1]: putting lock file in `sync.sh` means if u try and run it manually and there is a cron running it exits gracefully.

**Comment by Johanndutoit** [comment]
@bruskiza @smn do me a favour and go through setup.sh from this PR / branch. Thought of something this morning that might have caused issues. 

Only known keys were being stopped, so the old captive would have kept running when we remove it from the list.

Updated the setup.sh script to kill all containers and then start the listed apps.

Would just love some feedback / double checks if the logic makes sense - @jonathanendersby @smn @bruskiza

**Pull request merged**